### PR TITLE
feat: sept release notes

### PR DESCRIPTION
**Pull request by KaustubhKumar05**

### CODE REVIEW

**Review by ygit — APPROVED** [review]

File: docs/javascript/v2/release-notes/release-notes.mdx [hunk 1]
```diff
@@ -16,6 +16,23 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/hms-virtual-background      | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-virtual-background.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-virtual-background)           |
 
 
+## 2024-09-30
+Released: `@100mslive/hms-video-store@0.12.21`, `@100mslive/react-sdk@0.10.21`, `@100mslive/hls-player@0.3.21`, `@100mslive/roomkit-react@0.3.21`, `@100mslive/hms-whiteboard@0.0.11`, `@100mslive/hms-virtual-background@1.13.21`
+
+### Fixed:
+- Roomkit Prebuilt: HLS player takes more space in fullscreen mode
```
Comment (raviteja83) [inline docs/javascript/v2/release-notes/release-notes.mdx:23]: Make HLS Player take more space in fullscreen mode

File: docs/javascript/v2/release-notes/release-notes.mdx [hunk 2]
```diff
@@ -16,6 +16,23 @@ description: Release Notes for 100ms JavaScript SDK
...
+- Roomkit Prebuilt: Device label categorization
```
Comment (raviteja83) [inline docs/javascript/v2/release-notes/release-notes.mdx:24]: this is broken I think. Also the message should be just the icon not showing correct device. this does not tell anything

File: docs/javascript/v2/release-notes/release-notes.mdx [hunk 3]
```diff
@@ -16,6 +16,23 @@ description: Release Notes for 100ms JavaScript SDK
...
+- Avatar showing when returning from background in mweb
```
Comment (raviteja83) [inline docs/javascript/v2/release-notes/release-notes.mdx:25]: Avatar shown for a split second when vb is enabled in mweb.

File: docs/javascript/v2/release-notes/release-notes.mdx [hunk 4]
```diff
@@ -16,6 +16,23 @@ description: Release Notes for 100ms JavaScript SDK
...
+- Linear retry timeout for reconnection
+- Roomkit Prebuilt: Feedback submission endpoint
```
Comment (raviteja83) [inline docs/javascript/v2/release-notes/release-notes.mdx:27]: we can omit this.

File: docs/javascript/v2/release-notes/release-notes.mdx [hunk 5]
```diff
@@ -16,6 +16,23 @@ description: Release Notes for 100ms JavaScript SDK
...
 
 
+## 2024-09-30
+Released: `@100mslive/hms-video-store@0.12.21`, `@100mslive/react-sdk@0.10.21`, `@100mslive/hls-player@0.3.21`, `@100mslive/roomkit-react@0.3.21`, `@100mslive/hms-whiteboard@0.0.11`, `@100mslive/hms-virtual-background@1.13.21`
+
+### Fixed:
+- Roomkit Prebuilt: HLS player takes more space in fullscreen mode
+- Roomkit Prebuilt: Device label categorization
+- Avatar showing when returning from background in mweb
+- Linear retry timeout for reconnection
```
Comment (raviteja83) [inline docs/javascript/v2/release-notes/release-notes.mdx:26]: FYI, this is not internal release notes. makes no sense for the client. 

`Retry timeout not happening for 60s in some cases`

File: docs/javascript/v2/release-notes/release-notes.mdx [hunk 6]
```diff
@@ -16,6 +16,23 @@ description: Release Notes for 100ms JavaScript SDK
...
+- Roomkit Prebuilt: Feedback submission endpoint
+- Video not recovering after degradation gets fixed
```
Comment (raviteja83) [inline docs/javascript/v2/release-notes/release-notes.mdx:28]: In some cases, video is not getting recovered after degradation.

**Pull request merged**